airbase/pom.xml

```diff
@@ -738,7 +739,7 @@
                     </dependencies>
                     <configuration>
                         <skip>${air.check.skip-license}</skip>
-                        <skipExistingHeaders>true</skipExistingHeaders>
+                        <skipExistingHeaders>${air.license.skip-existing-headers}</skipExistingHeaders>
```

Comment:
Why not always `false`?

Reply:
(it's `true` since ever (d71d25e711593aff2a1e339d146d8a83205e2996))

Reply:
Just to preserve compatibility. IMO it should be false.

Reply:
@electrum can you please chime in here?

Reply:
Let’s make this false, since that is the better default. Any projects that need to skip can set the property when updating.

Reply:
Updated default to true and adjusted CHANGES entry accordingly.